vm/object.go

```diff
@@ -22,26 +22,30 @@ func (p *Pointer) returnClass() *RClass {
 
 // RObject represents any non built-in class's instance.
 type RObject struct {
-	Class             *RClass
+	*baseObj
+	InitializeMethod *MethodObject
+}
+
+type baseObj struct {
```

Comment:
By embedding `baseObj`, every object can have consist attributes like class and instance variables.